data-prepper-plugins/ocsf/src/main/resources/org/opensearch/dataprepper/transforms/templates/ocsf-v1.1-panw-traffic-template.yaml

```diff
@@ -0,0 +1,673 @@
+                  drop: Close
+                  drop ICMP: Close
+                  reset both: Reset
+                  reset client: Reset
+                  reset server: Reset
+    - translate:
+        mappings:
+          - source: Action
+            targets:
+              - target: action_id
+                default: 0
+                map:
+                  allow: 1
+                  deny: 2
+                  drop: 99
+                  drop ICMP: 99
+                  reset both: 99
+                  reset client: 99
+                  reset server: 99
+    - translate:
+        mappings:
+          - source: Action
+            targets:
+              - target: action
+                default: Other
+                map:
+                  allow: Allowed
+                  deny: Denied
+                  drop: Other
+                  drop ICMP: Other
+                  reset both: Other
+                  reset client: Other
+                  reset server: Other
+    - translate:
+        mappings:
+          - source: Action
+            targets:
+              - target: type_uid
+                default: 400199
+                map:
+                  allow: 400101
+                  deny: 400105
+                  drop: 400102
+                  drop ICMP: 400102
+                  reset both: 400103
+                  reset client: 400103
+                  reset server: 400103
+    - translate:
+        mappings:
+          - source: Action
+            targets:
+              - target: type_name
+                default: 'Network Activity: Unknown'
+                map:
+                  allow: 'Network Activity: Open'
+                  deny: 'Network Activity: Refuse'
+                  drop: 'Network Activity: Close'
+                  drop ICMP: 'Network Activity: Close'
+                  reset both: 'Network Activity: Reset'
+                  reset client: 'Network Activity: Reset'
+                  reset server: 'Network Activity: Reset'
+    - translate:
+        mappings:
+          - source: Action
+            targets:
+              - target: status_id
+                default: 0
+                map:
+                  allow: 1
+                  deny: 2
+                  drop: 99
+                  drop ICMP: 99
+                  reset both: 99
+                  reset client: 99
+                  reset server: 99
+    - translate:
+        mappings:
+          - source: Action
+            targets:
+              - target: status
+                default: 0
+                map:
+                  allow: Success
+                  deny: Failure
+                  drop: Other
+                  drop ICMP: Other
+                  reset both: Other
+                  reset client: Other
+                  reset server: Other
+    - rename_keys:
+        entries:
+          - from_key: Action
+            to_key: unmapped/Action
+            overwrite_if_to_key_exists: true
+    - convert_type:
```

Comment:
It may be good to change these to:

```
convert_type:
  keys: [status_id, action_id, type_uid, ...]
  type: integer
```

But, we can address this later.

Reply:
Yeah. I took the yaml config given by security lake engineer as is. We have identified some changes that can be done. We can do that in a separate PR.